src/index.js

```diff
@@ -46,7 +46,13 @@ export default function fetch(url, opts) {
 		const request = new Request(url, opts);
 		const options = getNodeRequestOptions(request);
 
-		const send = (options.protocol === 'https:' ? https : http).request;
+    const protocol = (options.protocol === 'https:' ? https : http)
```

Comment:
Indentation seems of here

```suggestion
		const protocol = (options.protocol === 'https:' ? https : http)
```